### ISSUE TITLE
Can't open openload/ol.link

### ISSUE THREAD

**Issue by techsword11** [reek/anti-adblock-killer#2854]
I cant open link https://ol.link/f/OnJBmSvv5Xk/P.17.DP3.0.rar .
i use chrome, Adblock ( using Acceptable Ad, AdBlock custom filter, EasyList, Adblock Warning Removal list, EasyPrivacy, Malware protection, AakList (Anti-Adblock Killer), Adware filters), Tampermonkey+adbypasser+anti adblocker
Proof:
![screenshot 8](https://cloud.githubusercontent.com/assets/25822171/23024582/879da732-f48d-11e6-9803-591f21e002ea.png)
Manual Filter:
![screenshot 9](https://cloud.githubusercontent.com/assets/25822171/23024595/9130f72c-f48d-11e6-851f-0c3472495b5b.png)

**Comment by leentorenvliet** [comment]
Firefox says it's a deceptive site. Better leave it closed maybe.

**Comment by techsword11** [comment]
@leentorenvliet No, it is alternative link from openload. I cant open openload because my isp block it

**Comment by uBlock-user** [comment]
If you're using uBO, then add

````
ol.link,openload.co##script:inject(abort-on-property-read.js, adblock2)
ol.link,openload.co##script:inject(setTimeout-defuser)

**Comment by techsword11** [comment]
@uBlock-user i not using uBO btw
